### ISSUE TITLE
More environments, trees and widgets!

### ISSUE THREAD

**Issue by panagiotouandrew** [bk20dev/forest#20]
It would be great if we had multiple "environments" instead of the green one that always holds our trees in the Forest view. (e.g. snowy for winter, beach like on summer). Another thing that I find pretty important is having widgets. They would be very useful, because since the app launched we only had a notification. Not the best thing. anyways, I hope you find these useful and implement them in the feature :)

**Comment by dilinzel-ine** [comment]
Why isn't the app getting any updates? Please can anyone let me know. 🥺

**Comment by bk20dev** [comment]
Hi! Sadly, I don't have the time to update the app right now and I can't promise when I'll be able to implement all the features you've suggested. Since October, I've been busy with my university studies and they're quite harsh at the moment.
In September, I've made some vital changes on a separate branch, but I'm like 50% there. I'd prefer not to make any other updates until these changes are complete, because they impact the way the steps are counted. It essentially requires me to rewrite most parts of the app, which I don't have the time for.
The first university term ends in February, and I might have some time to complete the rewrite then, but I don't want to make any promises. I’m truly sorry if I’ve disappointed anyone, but I need to prioritize my studies for now.

**Comment by N-U-LL** [comment]
any updates on this ?

**Comment by bartek537** [comment]
Yes and no. I made some changes to the step counter service locally in late September this year, but I haven’t pushed them yet. I still have some work to do, but I think I’m closer to finishing it this time. Once it’s done, I’ll be ready to work on this and other updates. Stay tuned!

**Comment by xSnowLeopardx** [comment]
> Yes and no. I made some changes to the step counter service locally in late September this year, but I haven’t pushed them yet. I still have some work to do, but I think I’m closer to finishing it this time. Once it’s done, I’ll be ready to work on this and other updates. Stay tuned!

No rush! Still enjoy the app a lot.

**Comment by MahmoudCmaher** [comment]
Any updates? 
I'm using the app via my Redmi 9T, but it doesn't work at all (although I set the battery preference to "no restrictions")
